iidm/iidm-modification/src/main/java/com/powsybl/iidm/modification/scalable/DanglingLineScalable.java

```diff
@@ -166,4 +164,15 @@ public double maximumValue(Network n) {
     public double minimumValue(Network n) {
         return minimumValue(n, scalingConvention);
     }
+
+    @Override
+    public double getCurrentPower(Network network, ScalingConvention scalingConvention) {
```

Comment:
Current power is not the good term because we have also the `current` in powsybl that means intensity. So we have to find something else. Note that in CGMES, we call that steady state hypothesis.

Reply:
Maybe just `getPower`.

Reply:
I propose "getOngoingPower" in order to stay more precise than "getPower"